### PR TITLE
MAT-5645: Connecting execution test results to actual display

### PR DESCRIPTION
**Pull request by gregory-akins**
## MADiE PR

Jira Ticket: [MAT-5645](https://jira.cms.gov/browse/MAT-5645)
(Optional) Related Tickets:

### All Submissions
* [X] This PR has the JIRA linked.
* [X] Required tests are included
* [X] No extemporaneous files are included (i.e Complied files or testing results)
* [X] This PR is into the **correct branch**.
* [X] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/editTestCase/qdm/EditTestCase.tsx [hunk 1]
```diff
@@ -244,11 +258,48 @@ const EditTestCase = () => {
...
+          //Set's an entry = IPP & numeric value from results
+          populationMap.set(value[1], eval(`results[0].${value[0]}`));
+        });
+
+        groupsMap.set("" + pop, populationMap);
+
+        currentTestCase.groupPopulations.forEach((value) => {
+          if (value.groupId === pop) {
+            value.populationValues.forEach((population) => {
+              //console.log(`Group Population Values for ${value.groupId}`, JSON.stringify(population.name, null, 2));
```
Comment (sb-prateekkeerthi) [inline src/components/editTestCase/qdm/EditTestCase.tsx]: I think we need to delete this console.log

**Review by sb-prateekkeerthi — CHANGES_REQUESTED** [review]

File: src/components/editTestCase/qdm/EditTestCase.tsx (hunk 1 above)
Comment (gregory-akins) [reply src/components/editTestCase/qdm/EditTestCase.tsx]: Done

**Review by sb-cecilialiu — APPROVED** [review]

**Review by sb-prateekkeerthi — APPROVED** [review]
